ten-simple-rules-dockerfiles.Rmd

```diff
@@ -394,9 +394,9 @@ Consult Ten Simple Rules paper by Perez-Riverol et al. [@perez-riverol_ten_2016]
 # 10. Use the container daily, rebuild the image weekly {-}
 
 A container that is not regularly rebuilt risks not getting essential security updates, and serving as a risk for its users.
-If you are using a container built by a `Dockerfile` regularity in your work, it's recommended to set up continuous deployment, or a strategy to ensure that the container is regularly built and tested.
+If you are using a container built by a `Dockerfile` regularly in your work, it's recommended to set up continuous deployment, or a strategy to ensure that the container is regularly built and tested.
 While there is no hardened rule, it would be good practice to rebuild a container every once or two weeks that you use on a daily basis.
-In an idealistic world, containers that we built years ago should rebuild seamlessly, but this isn't the case, especially with rapidly changing technology relevant to machine learning and data science.
+In an idealistic world, containers that we built years ago should rebuild seamlessly, but this isn't neessarily the case, especially with rapidly changing technology relevant to machine learning and data science.
```

Comment:
nessarily -> necessarily

Reply:
Fixed that one because it seemed obvious.